tests/test_neuron_models.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+import h5py
+import bluepysnap.neuron_models as test_module
+from bluepysnap.circuit import Circuit, Config
+from bluepysnap.sonata_constants import Node
+from bluepysnap.exceptions import BluepySnapError
+
+import pytest
+
+from utils import TEST_DATA_DIR, copy_circuit, copy_config, edit_config, create_node_population
+
+
+def test_invalid_model_type():
+    with copy_circuit() as (circuit_copy_path, config_copy_path):
+        config = Config(config_copy_path).resolve()
+        nodes_file = config["networks"]["nodes"][0]["nodes_file"]
+        with h5py.File(nodes_file, 'r+') as h5f:
+            grp_name = 'nodes/default/0/model_type'
+            data = h5f[grp_name][:]
+            del h5f[grp_name]
+            h5f.create_dataset(grp_name,
+                               data=["virtual"] * data.shape[0],
+                               dtype=h5py.string_dtype())
+        nodes = create_node_population(nodes_file, "default")
+        with pytest.raises(BluepySnapError) as e:
+            test_module.NeuronModelsHelper({}, nodes)
+        assert 'biophysical or point node population' in e.value.args[0]
+
+
+def test_get_invalid_node_id():
+    nodes = create_node_population(str(TEST_DATA_DIR / 'nodes.h5'), "default")
+    config = Config(TEST_DATA_DIR / 'circuit_config.json').resolve()
+    test_obj = test_module.NeuronModelsHelper(config['components'], nodes)
+
+    with pytest.raises(BluepySnapError) as e:
+        test_obj.get_filepath('1')
+    assert "node_id must be a int or a CircuitNodeId" in e.value.args[0]
+
+
+def test_get_filepath_biophysical():
+    nodes = create_node_population(str(TEST_DATA_DIR / 'nodes.h5'), "default")
+    config = Config(TEST_DATA_DIR / 'circuit_config.json').resolve()
+    test_obj = test_module.NeuronModelsHelper(config['components'], nodes)
+
+    node_id = 0
```

Comment:
Can you test with a CircuitId ? 
```
node_id = CircuitNodeId("default", 0)
```

Reply:
And maybe with 
```
node_id = CircuitNodeId("default2", 0)
```
Just to check what is returned (I most probably forgot to test this in the MorphHelper also)

Reply:
Nope, you didn't forget to test. I didn't test it because this check seemed to me outside of this context. Should not be enough of tests that check different usages of `CircuitNodeId` separately?

Reply:
> Should not be enough of tests that check different usages of CircuitNodeId separately?

Not sure to understand what you mean here.

Reply:
Then lets resolve as you requested.